addons/web_editor/static/src/js/wysiwyg/widgets/image_optimize_dialog.js

```diff
@@ -239,7 +239,7 @@ var ImageOptimizeDialog = Dialog.extend({
         var self = this;
         var name = this.$nameInput.val();
         var params = {
-            'name': name,
+            'name': `${name}.optimize`,
```

Comment:
Strange this commit, did not we decide to optimize by default on drop (or maybe this commit is doing that and only the commit message is misleading) ?

Reply:
Yes, this commit does several things, including optimize on drop but it's indeed missing from the commit message, I'll add that.